cyhy/mailer/__init__.py

```diff
@@ -1,3 +1,3 @@
 """This package contains the cyhy-mailer code."""
 
-__version__ = "1.4.1"
+__version__ = "1.4.2"
```

Comment:
I think this change classifies as a minor version bump since we're adding new functionality.

```suggestion
__version__ = "1.5.0"
```